### ISSUE TITLE
The requirements are not clealy defined 

### ISSUE THREAD

**Issue by mandar-gite** [Duane321/mutual_information#2]
I am not able to run the code . The modules such as '''mountain_car,  LargeRandomWalk ''' are missing  and not able to pip install them . Can you give the details of the enviornment to recreate and run these files

**Comment by Duane321** [comment]
Yes, it's not a package to be installed. It's just a collection of notebooks. Which notebook in particular are you trying to run?

**Comment by mandar-gite** [comment]
large_random_walk.ipynb is the name of notebook I am trying to run on colab/ my machine .
